clang/runtime/dpct-rt/include/dpct/graph.hpp

```diff
@@ -22,5 +22,31 @@ typedef sycl::ext::oneapi::experimental::command_graph<
     sycl::ext::oneapi::experimental::graph_state::executable>
     *command_graph_exec_ptr;
 
+static inline std::unordered_map<sycl::queue *, command_graph_ptr>
+    queue_graph_map;
+
+static bool command_graph_begin_recording(sycl::queue *queue_ptr) {
+  auto graph = new sycl::ext::oneapi::experimental::command_graph<
```

Comment:
I assume the `cudaGraphCreate` will be mapped to `command_graph_create` and `cudaGraphDestroy` will be mapped to `command_graph_destroy`.

1. Why the new operation is not in the `command_graph_create`?
2. Will you `delete` this `graph` variable in `command_graph_destroy`?

Reply:
Yes, `cudaGraphDestroy` will be used to delete the variable.

 I plan on removing `cudaGraphCreate` API because: 

- `cudaGraphCreate(&graph)` does not have the queue which needs to be given to `command_graph` on initialization, so we cannot create a new pointer by migrating `cudaGraphCreate`. 
- Its is an optional API (gromacs, tiny-cuda-nn do not use `cudaGraphCreate`).

Reply:
OK